YouTubeSearch/src/redux/actionType.ts

```diff
@@ -0,0 +1,19 @@
+export const HOME_VIDEOS_SUCCESS: string = 'HOME_VIDEOS_SUCCESS';

```

Comment:
Аналогично, имя action должно содержать название действия
```suggestion
export const FETCH_HOME_VIDEOS_SUCCESS: string = 'FETCH_HOME_VIDEOS_SUCCESS';
```